REPCO-SPEC.md

```diff
@@ -1,4 +1,4 @@
-# repco
+# repco [no longer up to date]
```

Comment:
Let's add a paragraph below the header like

*This specification is currently in review and will be updated soon. Please check the [doc](doc) folder for up-to-date descriptions of the data model.*

or something like that.